packages/cli/src/help/flag.ts

```diff
@@ -0,0 +1,78 @@
+import chalk from 'chalk'
+import { Command } from '@oclif/core'
+import { HelpFormatter } from '@oclif/core/lib/help/formatter'
+import { BaseCommand } from '@preevy/cli-common'
+
+const {
+  underline,
+} = chalk
+const {
+  dim,
+} = chalk
+
+export default class Flag extends HelpFormatter {
+  public getFlags():string {
+    const flags = Object.entries(BaseCommand.baseFlags)
+      .filter(([_name, flag]) => !flag.hidden)
+      .map(([name, flag]) => {
+        flag.name = name
+        return flag
+      })
+
+    const body = this.flagFormatter(flags)
+
+    return this.section('GLOBAL FLAGS', body)
+  }
+
+  protected flagHelpLabel(flag: Command.Flag.Any, showOptions = false): string {
```

Comment:
instead of copying the oclif code (which can change and break stuff), it's possible to inherit from the original `CommandHelp` to modify it and use it in the custom `Help` class. There is an example at the oclif tests: https://github.com/oclif/core/blob/c992403175c40492a5b724af3c117c6a059b4843/test/help/show-customized-help.test.ts#L14